packages/messenger-widget/src/hooks/auth/useAuth.ts

```diff
@@ -1,21 +1,31 @@
 /* eslint-disable no-console */
 import { useAccount, useWalletClient } from 'wagmi';
-
 import { Account } from '@dm3-org/dm3-lib-profile';
 import { UserDB } from '@dm3-org/dm3-lib-storage';
-import { useEffect, useMemo, useState } from 'react';
+import { useEffect, useMemo, useState, useContext } from 'react';
 import { useMainnetProvider } from '../mainnetprovider/useMainnetProvider';
 import { AccountConnector } from './AccountConnector';
 import {
     ConnectDsResult,
     DeliveryServiceConnector,
 } from './DeliveryServiceConnector';
+import { GlobalContext } from '../../utils/context-utils';
+import {
+    AccountsType,
+    Actions,
+    CacheType,
+    ConnectionType,
+    ModalStateType,
+    UiStateType,
+    UiViewStateType,
+} from '../../utils/enum-type-utils';
 import { useTopLevelAlias } from '../topLevelAlias/useTopLevelAlias';
 
 export const useAuth = (onStorageSet: (userDb: UserDB) => void) => {
     const { resolveAliasToTLD } = useTopLevelAlias();
     const { data: walletClient } = useWalletClient();
     const mainnetProvider = useMainnetProvider();
+    const { dispatch } = useContext(GlobalContext);
```

Comment:
dispatch is about to be removed and will be replaced by hooks over time. useAuth was the first hook and more about to follow after the state refactoring has been finished.
Hence please don't use dispatch anymore for any new developments. If you still have to perform an action within the legacy state management use a callback function similar to onStorageSet

Reply:
Ok